class AuthError(Exception):
    pass

class StorageError(Exception):
    pass

class LLMError(Exception):
    pass